torch2trt/converters/conv2d.py

```diff
@@ -0,0 +1,170 @@
+from torch2trt.torch2trt import *
+from torch2trt.module_test import add_module_test
+from .Conv2d import *
+
+@tensorrt_converter('torch.nn.functional.conv2d')
+def convert_conv2d(ctx):
+    in_channels  = ctx.method_args[0].size()[1]
+    out_channels = ctx.method_args[1].size()[0]
+    kernel_size  = tuple(ctx.method_args[1].size()[2:4])
+    stride       = ctx.method_args[3]
+    padding      = ctx.method_args[4]
+    dilation     = ctx.method_args[5]
+    groups       = ctx.method_args[6]
+    bias = False if ctx.method_args[2] is None else True
+
+    module = torch.nn.Conv2d(
```

Comment:
module refers to mapping of native pytorch layer to Tensorrt layer. 

Refer to this documentation

https://docs.nvidia.com/deeplearning/tensorrt/archives/tensorrt-601/tensorrt-api/python_api/infer/Graph/Layers.html#iconvolutionlayer

nn.conv2d should map to iconvolution layer in trt

Reply:
Very happy to receive your reply！
The torch.nn.Conv2d maps to iconvolution layer in trt, and it's mature.
I just wrapped torch.nn.functional.conv2d with torch.nn.Conv2d, they have the same effect, so I don't have to make wheels anymore.